src/shogun2-core/src/main/java/de/terrestris/shogun2/web/EndpointDocController.java

```diff
@@ -26,6 +26,7 @@
 	 * method-level @RequestMapping annotations in @Controller classes.
 	 */
 	@Autowired
+	@Qualifier("handlerMapping")
```

Comment:
Please rename the qualifier to something more meaningful, e.g. `requestHandlerMapping`.